__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const { TestWatcher } = require('jest');
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+//3
+describe('GET /api/topics task 3',() => {
+    test('responds with status code 200',() => {
+        return request(app).get('/api/topics')
+        .expect(200)
+    });
+    test('respond body will contain an array of topic objects',() => {
+        return request(app).get('/api/topics')
+        .then(({body}) => {
+            expect(Array.isArray(body.topics)).toBe(true);
```

Comment:
You don't need to explicitly test for an array here, this is implicitly implied after when you are checking the length and using a forEach